hail/python/hail/ir/matrix_reader.py

```diff
@@ -218,7 +218,8 @@ class TextMatrixReader(MatrixReader):
                       has_header=bool,
                       separator=str,
                       gzip_as_bgzip=bool,
-                      add_row_id=bool)
+                      add_row_id=bool,
+                      comment=sequenceof(str))
```

Comment:
depending on whether or not you really want to allow ["str1", "str2"], this could be str (although sequenceof also works)

Reply:
Reading further down it does seem you allow a list. My thought is that allowing either a single character or a regex is sufficient, and avoids needing to traverse (a small) list for every line. I think that operation could add up.

Reply:
This is an identical interface to `import_table`, where we added the possibility of multiple comment characters to provide better user interfaces. I don't think you'd be able to detect a performance cost here.

Reply:
I see, keeping interface same seems correct.